packages/@sanity/base/src/presence/overlay/RegionsWithIntersections.styled.tsx

```diff
@@ -0,0 +1,65 @@
+import styled, {css} from 'styled-components'
+import {WithIntersection} from './WithIntersection'
+
+interface styleProps {
+  debug: boolean
+  margins?: [number, number, number, number]
+}
+
+export const RootWrapper = styled.div`
+  position: relative;
+`
+
+export const OverlayWrapper = styled.div`
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  pointer-events: none;
+  z-index: 5;
```

Comment:
This `z-index` actually needs to be higher than 10, since `ChangeBar` makes each form field have `z-index: 10` on :hover.

PS. This is an old bug, so strictly not in scope in this PR. But maybe worth doing anyway?

![image](https://user-images.githubusercontent.com/406933/133583498-cc78e3a2-0476-4bd3-acac-aa4df45b96ef.png)

Reply:
Updated the z-index to be 13. I was doing some more tests and noticed that this also happened when the `ActivateOnFocus` overlay was active. That overlay (`ActivateOnFocus`) is at z-index 12, so this value I picked is the lowest that works.